src/fairchem/core/models/base.py

```diff
@@ -189,6 +189,10 @@ def no_weight_decay(self) -> list:
 
 
 class HeadInterface(metaclass=ABCMeta):
+    @property
+    def use_amp(self):
+        return False
```

Comment:
should the default be to not use it?

Reply:
When i talked to @wood-b he mentioned that we should have it off by default, it shouldnt change much memory usage and from our experiments it only helps with performance metrics